### PR TITLE
Civ 4439 bug

### PR DESCRIPTION
**Pull request by deepthidoppalapudihmcts**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Screenshot attached for the Evidence of manual testing
- [x] Functional, Smoke and API tests have been run locally
- [x] Full gradle build and run tests with coverage have been completed (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-4439

### Change description ###

Null pointer Exception when calling State flow engine  to evaluate data during createApplication event 

### Testing branch and instruction ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by smitatalawadekar-hmcts — APPROVED** [review]

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -10,6 +10,7 @@ def component = "general-applications"
 def camundaBranch = "master"
 def generalappCCDBranch = "sprint3integration"
 
+
```
Comment (karthick-mohanasundaram-HMCTS) [inline Jenkinsfile_CNP:13]: Could we revert this change if it's not intentional please?

Comment (deepthidoppalapudihmcts) [reply Jenkinsfile_CNP:13]: Yes we can do now

**Review by karthick-mohanasundaram-HMCTS — APPROVED** [review]
